.cirrus.yml

```diff
@@ -15,13 +15,19 @@ package_ci_task:
     cpu: 2
     memory: 12G
 
+  test_script:
+    - zkg test .
+
   install_script:
-    - zkg install --force .
+    - zkg install --force --skiptests .
```

Comment:
I am confused now why we install with `--force` if it in turn forces us to deviate from the installation method a user would default to (ideally we'd mirror what a user would do during installation as much as possible and only add our own steps _in addition to that_).

Could you add a comment somewhere here capturing the reasoning?

Reply:
"Force" avoids interactive prompts ("do you want to install package X?").

Reply:
Hmm, that's unfortunate. Could we maybe instead work around that with e.g.,

    $ yes | zkg install .
    $ echo Y | zkg install .

I think I tried using `yes` previously when this CI also had other issues, so maybe we need to work around issues with `echo Y`.

Not something super critical which needs to block this PR, but IMO still nice to have.

Reply:
not too tempted to try that :) I think the force is fine